docs/references.bib

```diff
@@ -492,22 +522,6 @@ @Book{            cassels1967algebraic
   mrclass       = {00.04 (10.00)}
 }
 
-@Article{         cassels1950,
```

Comment:
I think if you merge master the order of this file is now fixed

Reply:
Yes, thanks.